setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="celobox",
+    version="0.1a",
+    packages=find_packages(),
+    install_requires=[
+        'selenium',
+        'requests',
+        'enum34',
+        'pyyaml',
+    ],
+    entry_points={
+        'console_scripts': [
+            'phantompasswd = celobox.phantompasswd:main'
```

Comment:
YOLO, man. Why can't we just use a name that's been well-established for 40 years. How often do you actually even run `passwd` anyway?! I haven't run it in at least a few months, so it's irrelevant now.

(Yes, agreed. (-: )